src/main/java/fr/openent/minetest/cron/ShuttingDownWorld.java

```diff
@@ -0,0 +1,82 @@
+package fr.openent.minetest.cron;
+
+import fr.openent.minetest.core.constants.Field;
+import fr.openent.minetest.service.ServiceFactory;
+import fr.openent.minetest.service.WorldService;
+import fr.wseduc.webutils.Either;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Promise;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.entcore.common.controller.ControllerHelper;
+
+public class ShuttingDownWorld extends ControllerHelper implements Handler<Long>   {
+    private static final Logger log = LoggerFactory.getLogger(ShuttingDownWorld.class);
+    private final WorldService worldService;
+
+    public ShuttingDownWorld(ServiceFactory serviceFactory) {
+        this.worldService = serviceFactory.worldService();
+    }
+
+    @Override
+    public void handle(Long event) {
+        log.info("shutting down world launched");
+        worldService.getMongo(null, null, null, null, null, null,
+                        true, true, new JsonObject())
+                .compose(this::updateStatusAllWorld)
+                .onSuccess(res -> log.info("shutting down cron finish successfully"))
+                .onFailure(err -> {
+                    log.error("shutting down cron on failure", err);
+                });
```

Comment:
```suggestion
        log.info("[" + this.getClass().getSimpleName() + "] shutting down world launched");
        worldService.getMongo(null, null, null, null, null, null,
                        true, true, new JsonObject())
                .compose(this::updateStatusAllWorld)
                .onSuccess(res -> log.info("[" + this.getClass().getSimpleName() + "] shutting down cron finish successfully"))
                .onFailure(err -> log.error("[" + this.getClass().getSimpleName() + "] " +
                        "shutting down cron on failure: " + err.getMessage()));
```

Exemple de log + indentation